packages/estatico-sass/index.js

```diff
@@ -36,29 +47,23 @@ const schema = Joi.object().keys({
  * @param {object} env - Optional environment config, e.g. { dev: true }
  * @return {object}
  */
-const defaults = (env = {}) => {
-  const autoprefixer = require('autoprefixer');
-  const clean = require('postcss-clean');
-  const filterStream = require('postcss-filter-stream');
-
-  return {
-    src: null,
-    srcBase: null,
-    dest: null,
-    watch: null,
-    minifiedSuffix: '.min',
-    plugins: {
-      sass: {
-        includePaths: null,
-      },
-      clone: env.ci,
-      postcss: [
-        autoprefixer(),
-      ].concat(env.dev ? [] : filterStream(['**/*', '!**/*.min*'], clean())),
+const defaults = (env = {}) => ({
+  src: null,
+  srcBase: null,
+  dest: null,
+  watch: null,
+  minifiedSuffix: '.min',
+  plugins: {
+    sass: {
+      includePaths: null,
     },
-    logger: new Logger('estatico-sass'),
-  };
-};
+    clone: env.ci,
```

Comment:
is clone still necessary?

Reply:
yup. asked kasia about it and they need it in case they gotta debug something locally

Reply:
ah, now i see what you meant! removed it!